libraries/adaptive-expressions/src/memory/simpleObjectMemory.ts

```diff
@@ -50,7 +52,7 @@ export class SimpleObjectMemory implements MemoryInterface {
         }
 
         const parts: string[] = path
-            .split(/[.\[\]]+/)
+            .split(/[.[\]]+/)
```

Comment:
Is this `\` removal ok?, i think it will break because it is now getting escaped

Reply:
It's shown as an unnecessary scape character.